components/tidb_query_expr/src/impl_math.rs

```diff
@@ -460,6 +483,28 @@ pub fn truncate_int_with_uint(arg0: &Int, _arg1: &Int) -> Result<Option<Int>> {
     Ok(Some(*arg0))
 }
 
+#[inline]
+#[rpn_fn]
+pub fn truncate_uint_with_int(arg0: &Int, arg1: &Int) -> Result<Option<Int>> {
+    let x = arg0;
+    let d = arg1;
```

Comment:
Maybe we can remove this assignment, as they add extra burden for code reading.

Reply:
OK.

done